<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowBus</class>
 <widget class="QMainWindow" name="WindowBus">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Автобус</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox#MyGroupBox { 
     border: 2px solid gray; 
     border-radius: 3px; 
 } 

QGroupBox::title { 
    background-color: transparent;
     subcontrol-position: top left; /* position at the top left*/ 
     padding:2 13px;
 } </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>331</width>
      <height>111</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid gray;</string>
    </property>
    <property name="title">
     <string>Автобус</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border: 0px;
}</string>
     </property>
     <property name="text">
      <string>Маршрут:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border: 0px;
}</string>
     </property>
     <property name="text">
      <string>Номер:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelNumberRoute">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border: 0px;
}</string>
     </property>
     <property name="text">
      <string>195</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelNumberBus">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>60</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border: 0px;
}</string>
     </property>
     <property name="text">
      <string>АИ 951 В</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>401</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Информация о движении автобуса:</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>331</width>
      <height>111</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid gray;</string>
    </property>
    <property name="title">
     <string>Водитель</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLabel" name="labelDriverFLP">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>181</width>
       <height>17</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	border: 0px;
}</string>
     </property>
     <property name="text">
      <string>Пертов Иван Троянович</string>
     </property>
    </widget>
    <widget class="QGraphicsView" name="graphicsView_3">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>121</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>190</y>
      <width>551</width>
      <height>291</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Остановка</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Время по расписанию</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Фактическое время</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>716</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Действия</string>
    </property>
    <addaction name="actionCallDriver"/>
    <addaction name="actionCallBus"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Показать другие автобусы на данном маршруте</string>
   </property>
  </action>
  <action name="actionCallDriver">
   <property name="text">
    <string>Связаться с водителем</string>
   </property>
  </action>
  <action name="actionCallBus">
   <property name="text">
    <string>Сделать объявление оп громкой связи</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
